### PR TITLE
Kind integration feature

### PR DESCRIPTION
**Pull request by jbr**
~~based on #67~~ no longer based on #67

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -8,6 +8,7 @@ default-run = "divviup_api_bin"
 [features]
 default = []
 aggregator-api-mock = []
+kind-integration = []
```
Comment (tgeoghegan) [inline Cargo.toml:11]: "Running it in Kind" is a shorthand for what we (Divvi Up) would use this for, but I think we should name this so it better captures what it does. How about `stub-api-integrations`?

**Review by tgeoghegan — APPROVED** [review]
Just one bikeshed-y thing but I'm happy with how small this change is!

**Comment by tgeoghegan** [comment]
With this change, I can run `divviup-api` as `cargo run --features aggregator-api-mock,kind-integration` and then:

```
> curl -s -H "Accept: application/vnd.divviup+json;version=0.1" localhost:8080/api/users/me | python -m json.tool
{
    "email": "kind@test.example",
    "email_verified": true,
    "name": "kind user",
    "nickname": "kind",
    "picture": null,
    "sub": "",
    "updated_at": "+002023-05-02T21:51:35.913540658Z",
    "admin": false
}
```

👍🏻 works like a charm!

File: Cargo.toml (hunk 1 above)
Comment (jbr) [reply Cargo.toml:11]: This doesn't actually stub api integrations, though. It's more like "integration test mode" — the http clients are still configured to talk to the external internet

**Comment by tgeoghegan** [comment]
> We could also drop the content-type and accept requirements with this feature to make things more concise

I don't think that's necessary. Whatever client we use to hit the divviup-api API in the integration test is perfectly capable of handling those headers correctly.

**Pull request merged**